test/TransformSpec.hs

```diff
@@ -51,7 +51,8 @@ spec =
                    , CodeBlock ("",["frost:text:insert"],[]) ""]
       let pandoc = Pandoc nullMeta blocks
       -- when
-      let Right(Pandoc _ transformedBlocks) =  run $ runError $ transform [textPlugin "hello world!"] pandoc
+      let Right(Pandoc _ transformedBlocks) =
```

Comment:
How is this change related to this commit?

Reply:
No formatter in the project yet, so I have to do refactoring by hand :) Could be a separate commit though